recipes/import-assertions-to-attributes/src/workflow.ts

```diff
@@ -0,0 +1,128 @@
+import { styleText } from 'node:util';
+import { api } from '@codemod.com/workflow';
+import type { Api } from '@codemod.com/workflow';
+import type { Edit } from '@ast-grep/napi';
+
+// We run this codemod on all JS/TS files
+// including CJS, MJS, TS, and JSX files.
+const globPattern = '**/*.{cjs,mjs,js,jsx,?(d.)cts,?(d.)mts,?(d.)ts,tsx}';
+
+async function processModule(astGrep: Api['astGrep'] ) {
+	await astGrep({
+        rule: {
+            any: [
+				// cover `import { something } from './module.json' assert { type: 'json' };`
+				{ kind: 'import_statement', regex: "import[^;]+assert\\s*\\{" },
```

Comment:
What if there is a semi colon in the import `'some;thing'`

Reply:
updated it's handled !